src/sentry/search/snuba/executors.py

```diff
@@ -471,17 +471,44 @@ def better_priority_aggregation(
     aggregate_kwargs: Optional[PrioritySortWeights] = None,
 ) -> Sequence[str]:
     event_age_hours = "divide(now() - timestamp, 3600)"
-    event_score = 1
+    event_age_weight = 1
     min_score = 0.01
     max_pow = 16
     event_halflife_hours = 4  # halves score every 4 hours
-    aggregate_event_score = f"greatest({min_score}, sum(divide({event_score}, pow(2, least({max_pow}, divide({event_age_hours}, {event_halflife_hours}))))))"
+    aggregate_event_score = f"greatest({min_score}, sum(divide({event_age_weight}, pow(2, least({max_pow}, divide({event_age_hours}, {event_halflife_hours}))))))"
     issue_age_hours = "divide(now() - min(timestamp), 3600)"
-    issue_score = 1
+    issue_age_weight = 1
     issue_halflife_hours = 24 * 7  # issues half in value every week
-    aggregate_issue_score = f"greatest({min_score}, divide({issue_score}, pow(2, least({max_pow}, divide({issue_age_hours}, {issue_halflife_hours})))))"
+    aggregate_issue_score = f"greatest({min_score}, divide({issue_age_weight}, pow(2, least({max_pow}, divide({issue_age_hours}, {issue_halflife_hours})))))"
+
+    if aggregate_kwargs is None:
```

Comment:
This shouldn't be none, I set defaults [here](https://github.com/getsentry/sentry/blob/master/src/sentry/api/endpoints/organization_group_index.py#L163). Maybe those can just be 1 instead.